core/src/test/java/com/exadel/aem/toolkit/core/injectors/ChildInjectorTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.core.injectors;
+
+import com.exadel.aem.toolkit.core.injectors.models.TestModelChild;
+
+import com.exadel.aem.toolkit.core.lists.models.internal.ListItemModel;
+
+import io.wcm.testing.mock.aem.junit.AemContext;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ChildInjectorTest {
+
+    private static final String EXPECTED_RESOURCE_TYPE1 = "wcm/foundation/components/responsivegrid";
+    private static final String EXPECTED_RESOURCE_TYPE2 = "etoolbox-authoring-kit/lists/components/content/listItem";
+    private static TestModelChild testModel;
+
+    @Rule
+    public final AemContext context = new AemContext();
+
+    @Before
+    public void beforeTest() {
+        context.addModelsForClasses(TestModelChild.class);
+        context.registerInjectActivateService(new ChildInjector());
+        context.load().json("/com/exadel/aem/toolkit/core/injectors/childInjector.json", "/content");
+        ResourceResolver resolver = context.resourceResolver();
+        context.request().setResource(resolver.getResource("/content/jcr:content"));
+        testModel = context.request().adaptTo(TestModelChild.class);
+    }
+
+    @Test
+    public void testInjectChild() {
```

Comment:
(naming) Please note that all other test cases in `core` are named like `shouldDoSomething`